class.jetpack-options.php

```diff
@@ -153,13 +183,28 @@ public static function get_option( $name, $default = false ) {
 	 * @param string $name Option name
```

Comment:
We should probably be more clear in the phpdoc if it's the raw option name, or the arg passed into the class (unprefixed)